Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -66,6 +67,20 @@ def livecheck_formulae(formulae_to_check, args)
       @livecheck_strategy_names.freeze
 
       has_a_newer_upstream_version = false
+
+      if args.json?
+        total_formulae = if formulae_to_check == Formula
+          formulae_to_check.count
+        else
+          formulae_to_check.length
+        end
+        ohai "Generating JSON"
+        progress = ProgressBar.create(
+          total:         total_formulae,
+          progress_mark: "#",
+          format:        "%b %c/%C",
+        )
+      end
```

Comment:
```suggestion
        end

        ohai "Generating JSON"
        progress = ProgressBar.create(
          total:  total_formulae,
          format: " %t: |%b%i| %c/%C ",
        )
      end

```

What do you think about this format? It resolves a few issues that I was experiencing when testing it:

* Added an extra space at the start, so the first character is more visible (and adds a space at the end to mimic the leading space).
* Added an indicator of the start/end of the progress bar (`|`).
* Filled the incomplete space in the progress bar, so the finished/total numbers are aligned to the right.

Reply:
That's actually what I had before creating the PR (like the bars in my scripts), haha. The reason I changed it was to maintain some consistency and make it look similar to the `curl` download progress bar, which seems to take up `0.8 * Tty.width` (72 #s on completion and my terminal width is 90).

I like this format better, so I don't mind changing it.